test/quantization/test_quantized_op.py

```diff
@@ -1657,6 +1657,83 @@ def test_group_norm(self, batches, num_groups, channels_per_group,
             self.assertTrue(pct_diff < 1e-6)
             self.assertTrue(pct_diff_off_by_one < 0.01)
 
+    @given(shapes=hu.array_shapes(3, 5, 2, 32),
+           torch_type=st.sampled_from((torch.qint8, torch.quint8, torch.qint32)),
+           X_rand_scale=st.floats(0.01, 1e3),
+           Y_scale=st.floats(0.2, 2.6),
+           Y_zero_point=st.integers(0, 5))
+    def test_instance_norm(self, shapes, torch_type, X_rand_scale,
```

Comment:
Since these tests are almost identical can we write a single test that loops over different norm ops?

Reply:
will do this refactor in a separate follow-up